File: Backend/ApiUniversidad/ApiUniversidad/Models/DocentesPorCurso.cs
using System;
using System.Collections.Generic;

namespace ApiUniversidad.Models;

public partial class DocentesPorCurso
{
    public Guid Id { get; set; }

    public Guid IdCurso { get; set; }

    public Guid IdDocente { get; set; }

    public DateTime FechaAlta { get; set; }

    public virtual Curso IdCursoNavigation { get; set; } = null!;

    public virtual Docente IdDocenteNavigation { get; set; } = null!;
}

File: Frontend/ApiUniversidad/UniversidadAngular/src/app/models/docentes-por-curso.ts
import { Curso } from "./curso";
import { Docente } from "./docente";

export class DocentesPorCurso {

    id?: string;
    idCurso: string = '';
    idDocente: string = '';
    fechaAlta: Date = new Date();
    idDocenteNavigation: Docente = new Docente();
    idCursoNavigation: Curso = new Curso();

}
